### PR TITLE
Refactor | Mobile Ready Navigation

### PR DESCRIPTION
**Pull request by Da-Colon**
## Description
Updates the sidebar into a `Drawer` menu on smaller screens

## Notes
- using the 'md' breakpoint, which makes changes at 615px
- Renamed any reference to `sidebar` to `navigation`
- extracted and repurposed sidebar components to change on smaller screens
- added a few dev libraries to ensure that mobile connections work properly (NodeJS missing files)

## Followup PRs
1. Update Components in Proposals workflow for Mobile
2. Update Favorites for Mobile (Drawer)
3. Update Wallet Menu for Mobile (Drawer)

## Known Issues
- The font for the text isn't quite matching, working on a new release with updated fonts and cleaning up `Input` component

## Screenshots (if applicable)
localhost_3000_ (31)

localhost_3000_ (32)

localhost_3000_ (33)

localhost_3000_ (34)

### CODE REVIEW

**Review by herbig — CHANGES_REQUESTED** [review]
Looks sick @Da-Colon , a few things (all on Chrome on Android Pixel 3a xl):

- [x] there's weird extra white space at the bottom when scrolling down (not sure we can fix this easily, if it's a pain don't spend too much time on it)
- [ ] FAQ, Discord, Docs links don't open, the Fractal app screen just become blank
- [ ] rather than "en-US" can we display the "English (EN)" text here?
- [x] can we close the drawer when a user select the DAO from the search bar?

File: src/i18n/index.ts [hunk 1]
```diff
@@ -57,7 +57,7 @@ export const supportedLanguages = {
     proposalMetadata: PROPOSAL_METADATA_EN,
     transaction: TRANSACTION_EN,
     treasury: TREASURY_EN,
-    sidebar: SIDEBAR_EN,
+    navigation: NAVIGATION_EN,
```
Comment (herbig) [inline src/i18n/index.ts:60]: ah good call on renaming this

**Comment by herbig** [comment]
@Da-Colon on Desktop, the navigation has shifted a bit so it's no longer centered vertically

see

https://app.dev.fractalframework.xyz/#/daos/0xF332aAb56e00f7FDEdEDDC50033Ce5e2D0DD6d61

vs

https://deploy-preview-1013--fractal-framework-interface-dev.netlify.app/#/daos/0xF332aAb56e00f7FDEdEDDC50033Ce5e2D0DD6d61

**Review by tbwebb22 — APPROVED** [review]
LGTM

**Review by herbig — APPROVED** [review]
🚀🚀🚀

**Pull request merged**